lib/goals.js

```diff
@@ -267,6 +267,7 @@ function distanceXZ (dx, dz) {
  * - facing - the direction the player must be facing
  * - facing3D - boolean, facing is 3D (true) or 2D (false)
  * - half - 'top' or 'bottom', the half that must be clicked
+ * - LOS - true or false, should the bot have line of sight off the placement face. Default true.
```

Comment:
This option is confusing because it's not the definition of line of sight. For instance if you set it to false it would still prune faces that are occluded by an other block. I'm fine with this function not being optional, but if you want to make it an option please find a more appropriate name.